CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# 3.0.0
+
+* Supports Chef 13
```

Comment:
We do in the readme and metadata. This was meant to say: 'from this release onwards, chef 13 is supported'